rgauto.c

```diff
@@ -794,8 +794,8 @@ static int window_state_parse(glkunix_library_state_t state, glkunix_unserialize
         window_textbuffer_t *dwin = win_textbuffer_create(win);
         win->data = dwin;
 
-        glkunix_unserialize_int(entry, "buf_width", &dwin->width);
-        glkunix_unserialize_int(entry, "buf_height", &dwin->height);
+        glkunix_unserialize_int(entry, "buf_width", (int*) &dwin->width);
+        glkunix_unserialize_int(entry, "buf_height", (int*) &dwin->height);
```

Comment:
It's lines like these which should probably be serializing/unserializing doubles?